tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -682,56 +681,6 @@ describe(__filename, () => {
     );
   });
 
-  it('shows add-ons in header if no specific addonType translation found', () => {
-    const root = renderAddonsWithType({
-      addonType: ADDON_TYPE_OPENSEARCH,
-      multipleAuthors: false,
-    });
-
-    expect(root.find(AddonsCard)).toHaveProp(
-      'header',
-      `More add-ons by ${fakeAuthor.name}`,
-    );
-  });
-
-  it('shows add-ons in header if no specific addonType translation found without More text', () => {
-    const root = renderAddonsWithType({
-      addonType: ADDON_TYPE_OPENSEARCH,
-      showMore: false,
-      multipleAuthors: false,
-    });
-
-    expect(root.find(AddonsCard)).toHaveProp(
-      'header',
-      `Add-ons by ${fakeAuthor.name}`,
-    );
-  });
-
-  it('shows add-ons in header if no specific addonType found with multiple authors', () => {
-    const root = renderAddonsWithType({
-      addonType: ADDON_TYPE_OPENSEARCH,
-      multipleAuthors: true,
-    });
-
-    expect(root.find(AddonsCard)).toHaveProp(
-      'header',
-      'More add-ons by these developers',
-    );
-  });
-
-  it('shows add-ons in header if no specific addonType found with multiple authors and without More text', () => {
-    const root = renderAddonsWithType({
-      addonType: ADDON_TYPE_OPENSEARCH,
-      showMore: false,
-      multipleAuthors: true,
-    });
-
-    expect(root.find(AddonsCard)).toHaveProp(
-      'header',
-      'Add-ons by these developers',
-    );
-  });
```

Comment:
I think we want to keep these test cases. Could you replace `addonType: ADDON_TYPE_OPENSEARCH` with `addonType: 'unknown-type'` in the 4 test cases above?

Reply:
Yes. I added it back.

Reply:
Works fine.